app/resources/create.html.template

```diff
@@ -99,8 +94,15 @@
                   </span>
                 </div>
               {% endif %}
+
+          <div class="name section">
```

Comment:
How about using "subsection" here instead of "section"? This is a section inside another section, so it sounds natural to make it "subsection". Let me know if it's not rendered as expected.

Basic expected hierarchy is:
```
fields-table
  section
    subsection (optional)
      field
        label
        value
```